src/services/namex.services.ts

```diff
@@ -238,8 +238,8 @@ export default class NamexServices {
       const blob = new Blob([response.data], { type: 'application/pdf' })
 
       // IE doesn't allow using a blob object directly as link href, so use msSaveOrOpenBlob
-      if (window.navigator && window.navigator.msSaveOrOpenBlob) {
-        window.navigator.msSaveOrOpenBlob(blob)
+      if (window.navigator && window.navigator['msSaveOrOpenBlob']) {
+        window.navigator['msSaveOrOpenBlob'](blob, 'Name Request Results')
```

Comment:
I did this (as in our other UIs) to bypass the warning, "property 'msSaveOrOpenBlob' does not exist on type 'Navigator'."

Reply:
We can _probably_ remove this IE code in all UIs. (Next time we're in this area of code.)

Just the fact that the function no longer exists on `window.navigator` tells me IE support is deprecated/obsolete.